pages/api/kyc/validate.ts

```diff
@@ -0,0 +1,65 @@
+import {NextApiRequest, NextApiResponse} from "next";
+
+import models from "db/models";
+
+import findUserBySession from 'helpers/query/findUserBySession';
+
+import { kycApi } from "services/api";
+
+async function get(req: NextApiRequest, res: NextApiResponse) {
+  try {
+    const { session_id } = req.headers
+
+    const user = await findUserBySession(req)
+
+    if(!user)
+      return res.status(500).json('User not found')
```

Comment:
this needs to be 404, not found; Also, provide a actual `json` or change to `text()`